src/TimeoutMigrationTool.NHibernate.AcceptanceTests/TimeoutMigrationTool.NHibernate.AcceptanceTests.csproj

```diff
@@ -12,7 +12,7 @@
 
   <ItemGroup>
     <PackageReference Include="NServiceBus.AcceptanceTesting" Version="7.4.4" />
-    <PackageReference Include="NServiceBus.RabbitMQ" Version="5.2.0" />
+    <PackageReference Include="NServiceBus.RabbitMQ" Version="6.0.0" />
```

Comment:
Should the acceptance tests not run using the version of the downstream that could still have mixed mode timeouts? Customers on the newer versions would already have migrated so we wouldn't be testing against actual production versions.

Reply:
Discussed with Daniel, this is not a concern